feat: ui flags in cloud native demo (#295)
open-feature/openfeature.dev: docs/tutorials/ofo.md

And finally, let's install the operator itself:

```shell

> kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.5.1/release.yaml && \

Reviewer: ```suggestion
kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.5.2/release.yaml && \
```

![k9s](@site/static/img/tutorials/k9s.png)

#### Forward the service (if not using kind)

> ⚠️ If you're using the [supplied `kind` config](#using-kind), ***you can skip to [Experiment with OpenFeature](#experiment-with-openfeature), this port is already forwarded.***

Author: ```suggestion
⚠️ If you're using the [supplied `kind` config](#using-kind), ***you can skip to [Experiment with OpenFeature](#experiment-with-openfeature), the ports are already forwarded.***
```
